scanner.go

```diff
@@ -36,6 +36,7 @@ import (
 // Scanner contains a YARA scanner
 type Scanner struct {
 	*scanner
+	rules *Rules
```

Comment:
There should be a comment that explains that the rules object might be garbage-collected and cause the underlying YR_RULES struct to be destroyed if this field were missing.

Reply:
Ack, will do!